CODEOWNERS

```diff
@@ -7,6 +7,7 @@
 # Add-on maintainers:
 /bundles/org.openhab.binding.airquality/ @kubawolanin
 /bundles/org.openhab.binding.airvisualnode/ @3cky
+/bundles/org.openhab.binding.alarmdecoder/ @bobadair @billfor @berndpfrommer
```

Comment:
Did @berndpfrommer recently work on this binding? Since the CODEOWNERS is used to request reviewers, we should stick to those people that recenetly did major work on that code.

Reply:
He is the author of the V1 binding, which this is derived from. I know he said he was short of time to work on an OH2 version, so I'm not sure if he even wants to see review notifications regarding it.  @berndpfrommer, do you want to be on this list?